libqtile/widget/memory.py

```diff
@@ -19,25 +18,25 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 from __future__ import division
+
+import psutil
+
 from libqtile.widget import base
 
 
 def get_meminfo():
     val = {}
-    with open('/proc/meminfo') as file:
-        for line in file:
-            key, tail = line.split(':')
-            uv = tail.split()
-            val[key] = int(uv[0]) // 1000
-    val['MemUsed'] = val['MemTotal'] - val['MemFree']
+    val['MemUsed'] = int(psutil.virtual_memory().used / 1024 / 1024)
+    val['MemTotal'] = int(psutil.virtual_memory().total / 1024 / 1024)
+    val['MemFree'] = val['MemTotal'] - val['MemUsed']
     return val
 
 
 class Memory(base.InLoopPollText):
     """Displays memory usage"""
     orientations = base.ORIENTATION_HORIZONTAL
     defaults = [
-        ("fmt", "{MemUsed}M/{MemTotal}M", "see /proc/meminfo for field names")
+        ("fmt", "{MemUsed}M/{MemTotal}M", "Formatting for field names")
```

Comment:
I've made them the same as the /proc interface. I'll put them in the docs

Reply:
My sentence was a continuation to:

> Maybe we could accept that config breakage but then expose all the stuff contained in psutil.virtual_memory(), so that people actually can replace what they used to do? We could then drop the useless mapping in names.

Sorry if that wasn't clear!

Reply:
Oh yeah that. Well, we could use these as general purpose widgets for portability (most people just want CPU/Mem/Swap usage anyway), and maybe keep the old ones for OS-specific hackery? Maybe rename the original one to linux_memory.py (since it is really OS-specific).